addon_service/management/commands/migrate_authorized_account.py

```diff
@@ -233,55 +233,59 @@ def migrate_for_user(
         )
         if not users_external_accounts:
             return
-        osf_account: ExternalAccount = users_external_accounts[0]
         user = OsfUser.objects.get(pk=user_settings.owner_id)
-        try:
-            credentials = self.get_credentials(external_service, osf_account)
-        except CredentialException as e:
-            logger.error(
-                f"Skipping account migration with {osf_account.pk=} for service {external_service.display_name} "
-                f"due to credentials parse error {e=}"
+        account_map = {}
+        for osf_account in users_external_accounts:
+            try:
+                credentials = self.get_credentials(external_service, osf_account)
+            except CredentialException as e:
+                logger.error(
+                    f"Skipping account migration with {osf_account.pk=} for service {external_service.display_name} "
+                    f"due to credentials parse error {e=}"
+                )
+                return
```

Comment:
This should likely be `continue` now that it may need to check multiple `ExternalAccount`s.